Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+        jQuery(subject).on("change", deleteCheckboxSelector, function () {
+
             Admin.switch_inline_form_errors(jQuery(this));
+
         });
+
     },
 
     /**
      * Disable inline form errors when the row is marked for deletion
      */
-    switch_inline_form_errors: function(subject) {
-        Admin.log('[core|switch_inline_form_errors] switch_inline_form_errors', subject);
+    "switch_inline_form_errors" (subject) {
 
-        var row = subject.closest('.sonata-ba-field-inline-table'),
-            errors = row.find('.sonata-ba-field-error-messages')
-        ;
+        Admin.log("[core|switch_inline_form_errors] switch_inline_form_errors", subject);
 
-        if (subject.is(':checked')) {
-            row
-                .find('[required]')
-                .removeAttr('required')
-                .attr('data-required', 'required')
-            ;
+        let row = subject.closest(".sonata-ba-field-inline-table"),
+            errors = row.find(".sonata-ba-field-error-messages");
+        if (subject.is(":checked")) {
 
+            row.
+                find("[required]").
+                removeAttr("required").
+                attr("data-required", "required");
             errors.hide();
+
         } else {
-            row
-                .find('[data-required]')
-                .attr('required', 'required')
-            ;
 
+            row.
+                find("[data-required]").
+                attr("required", "required");
             errors.show();
+
         }
+
     },
 
-    setup_tree_view: function(subject) {
-        Admin.log('[core|setup_tree_view] setup tree view', subject);
+    "setup_tree_view" (subject) {
+
+        Admin.log("[core|setup_tree_view] setup tree view", subject);
+
+        jQuery("ul.js-treeview", subject).treeView();
 
-        jQuery('ul.js-treeview', subject).treeView();
     },
 
     /** Return the width for simple and sortable select2 element **/
-    get_select2_width: function(element){
-        var ereg = /width:(auto|(([-+]?([0-9]*\.)?[0-9]+)(px|em|ex|%|in|cm|mm|pt|pc)))/i;
+    "get_select2_width" (element) {
+
+        const ereg = /width:(auto|(([-+]?([0-9]*\.)?[0-9]+)(px|em|ex|%|in|cm|mm|pt|pc)))/i;
 
-        // this code is an adaptation of select2 code (initContainerWidth function)
-        var style = element.attr('style');
-        //console.log("main style", style);
+        // This code is an adaptation of select2 code (initContainerWidth function)
+        let style = element.attr("style");
+        // Console.log("main style", style);
 
         if (style !== undefined) {
-            var attrs = style.split(';');
 
-            for (var i = 0, l = attrs.length; i < l; i = i + 1) {
-                var matches = attrs[i].replace(/\s/g, '').match(ereg);
-                if (matches !== null && matches.length >= 1)
+            const attrs = style.split(";");
+
+            for (let i = 0, l = attrs.length; i < l; i += 1) {
```

Comment:
- eslint: Identifier name 'i' is too short (< 2).
- eslint: Identifier name 'l' is too short (< 2).
- eslint: No magic number: 1.